sqlglot/dialects/starrocks.py

```diff
@@ -35,7 +35,10 @@ def _parse_unnest(self, with_alias: bool = True) -> t.Optional[exp.Unnest]:
             if unnest:
                 alias = unnest.args.get("alias")
 
-                if alias and not alias.args.get("columns"):
+                if not alias:
+                    alias = exp.TableAlias(columns=[exp.to_identifier("unnest")])
+                    unnest.set("alias", alias)
+                if not alias.args.get("columns"):
```

Comment:
These branches are independent now i.e if we hit the first then we'll not hit the second. Lets connect them:

 ```
if not alias:
  ...
elif not alias.args.get("columns"):
  ...
```

Reply:
Got it. I will fix it.